web/client/components/ScaleBar/ScaleBar.jsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2015, Marko Polovina
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+var React = require('react');
+
+/**
+ * [ScaleBarController new React Controller for ScaleBar]
+ */
+var ScaleBarController = React.createClass({
```

Comment:
Component name should be equal to file name. Moreover this is not a controller, but a Component. In general ScaleBar is a good name.